ts/components/services/ServicesSectionsList.tsx

```diff
@@ -0,0 +1,161 @@
+/**
+ * A component to render a list of services organized in sections, one for each organization.
+ */
+import { Option } from "fp-ts/lib/Option";
+import I18n from "i18n-js";
+import { Button, Text, View } from "native-base";
+import React, { ComponentProps } from "react";
+import { Image, NativeScrollEvent, NativeSyntheticEvent } from "react-native";
+import { StyleSheet } from "react-native";
+import customVariables from "../../theme/variables";
+import IconFont from "../ui/IconFont";
+import ServiceList from "./ServiceList";
+import ServiceSectionListComponent from "./ServiceSectionListComponent";
+
+type AnimatedProps = {
+  animated?: {
+    onScroll: (_: NativeSyntheticEvent<NativeScrollEvent>) => void;
+    scrollEventThrottle?: number;
+  };
+  paddingForAnimation: boolean;
+  AnimatedCTAStyle?: any;
+};
+
+type BaseProps = AnimatedProps &
+  ComponentProps<typeof ServiceSectionListComponent>;
+
+interface LocalProps extends BaseProps {
+  type: "Local";
+  onChooserAreasOfInterestPress: () => void;
+  organizationsFiscalCodesSelected: Option<Set<string>>;
+}
+
+interface OtherProps extends BaseProps {
+  type: "Other";
```

Comment:
```suggestion
  type: "All";
```

Reply:
It's not properly correct. Other is uses for both the "National" and the new "All" content. Maybe a `"Generic"` label can be more descriptive? Within 3e8dd55e69564af5309be09820d64059dd82cc3c I updated also the interface as `LocalSectionProps` and `GenericSectionProps`to be more descriptive

Reply:
Maybe I'm missing out: "**All**" sections contains all services (_National_ + _Locals_) and that ones we can't know if they are local or national (no metadata, or network fails)?
Anyway this suggestion aims to keep related code naming with what we show to the users.

Reply:
We can keep _Generic_ :+1: